### PR TITLE
VZ-7380: Update workload API content

### PR DESCRIPTION
**Pull request by bobdonat**
The pull request contains updates to the API content for the workload APIs.

### CODE REVIEW

File: platform-operator/helm_config/charts/verrazzano-application-operator/crds/oam.verrazzano.io_verrazzanocoherenceworkloads.yaml [hunk 1]
```diff
@@ -36,27 +36,25 @@ spec:
           metadata:
             type: object
           spec:
-            description: VerrazzanoCoherenceWorkloadSpec wraps a Coherence resource.
-              The Coherence object specified in the template must contain a spec field
-              and it may include a metadata field.
+            description: The desired state of a Verrazzano Coherence workload.
             properties:
               template:
+                description: The metadata and spec for the underlying <a href="https://oracle.github.io/coherence-operator/docs/3.2.8/#/docs/about/04_coherence_spec">Coherence</a>
```
[the remark this answers is not in the document]
Comment (bobdonat) [reply platform-operator/helm_config/charts/verrazzano-application-operator/crds/oam.verrazzano.io_verrazzanocoherenceworkloads.yaml:42]: yes, I can change to use that if you want.  I had changed it to be the same as the version we install.

File: application-operator/apis/oam/v1alpha1/verrazzanocoherenceworkload_types.go [hunk 2]
```diff
@@ -8,36 +8,40 @@ import (
...
+	// The desired state of a Verrazzano Coherence workload.
+	Spec VerrazzanoCoherenceWorkloadSpec `json:"spec,omitempty"`
+	// The observed state of a Verrazzano Coherence workload.
 	Status VerrazzanoCoherenceWorkloadStatus `json:"status,omitempty"`
 }
 
 // +kubebuilder:object:root=true
 
-// VerrazzanoCoherenceWorkloadList contains a list of VerrazzanoCoherenceWorkload
+// VerrazzanoCoherenceWorkloadList contains a list of VerrazzanoCoherenceWorkload resources.
```
Comment (MelwynPaul) [inline application-operator/apis/oam/v1alpha1/verrazzanocoherenceworkload_types.go:44]: Update: 
VerrazzanoCoherenceWorkload resources > Verrazzano Coherence workload resources

**Review by MelwynPaul — COMMENTED** [review]
Looks good to me.
Just have one comment.

File: platform-operator/helm_config/charts/verrazzano-application-operator/crds/oam.verrazzano.io_verrazzanocoherenceworkloads.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (bobdonat) [reply platform-operator/helm_config/charts/verrazzano-application-operator/crds/oam.verrazzano.io_verrazzanocoherenceworkloads.yaml:42]: fixed

File: application-operator/apis/oam/v1alpha1/verrazzanocoherenceworkload_types.go (hunk 2 above)
Comment (bobdonat) [reply application-operator/apis/oam/v1alpha1/verrazzanocoherenceworkload_types.go]: fixed

**Review by MelwynPaul — APPROVED** [review]

**Pull request merged**